pkg/trace/config/testdata/full.yaml

```diff
@@ -36,6 +36,10 @@ apm_config:
     - /health
     - /500
 
+  filter_tags:    
```

Comment:
Adding this here has no effect if you don't add the assertion into the test too: https://github.com/DataDog/datadog-agent/blob/fcc68a02c6c146826a302db9b39df3c761f7aac3/pkg/trace/config/config_test.go#L135

Reply:
They should be here:
https://github.com/DataDog/datadog-agent/blob/eedb2b483d891a837c2f3bed1d00a3e3a95b339f/pkg/trace/config/config_test.go#L184-L185